state/src/item/account.rs

```diff
@@ -131,16 +104,15 @@ impl Encodable for Account {
         s.append(&PREFIX);
         s.append(&self.balance);
         s.append(&self.seq);
-        s.append(&self.regular_key);
```

Comment:
You must decrease the number of the list.

Reply:
I modified it.